addon/ng2/blueprints/ng2/files/src/tsconfig.json

```diff
@@ -9,10 +9,16 @@
     "moduleResolution": "node",
     "noEmitOnError": true,
     "noImplicitAny": false,
-    "outDir": "../dist/",
+    "//outDir": "this option is used only during manual invocation of tsc usually while debugging",
+    "outDir": "../dist-manual/",
```

Comment:
Please keep this as `../dist` as this is for tooling, more than command line usage of `tsc`.